conf/nginx/nginx.conf.web.https.default.template

```diff
@@ -1,12 +1,33 @@
 # HTTPS Proxy Default Configuration
 #
+server {
+    ${core.ipboth.enabled}listen                  [::]:${web.https.port} default_server ipv6only=off;
+    ${core.ipv4only.enabled}listen                ${web.https.port} default_server;
+    ${core.ipv6only.enabled}listen                [::]:${web.https.port} default_server;
+
+    ssl                     on;
+    ssl_protocols           ${web.ssl.protocols};
+    ssl_prefer_server_ciphers ${web.ssl.preferserverciphers};
+    ssl_session_cache       ${ssl.session.cachesize};
+    ssl_session_timeout     ${ssl.session.timeout};
+    ssl_ciphers             ${web.ssl.ciphers};
+    ssl_ecdh_curve          ${web.ssl.ecdh.curve};
+    ssl_certificate         ${ssl.crt.default};
+    ssl_certificate_key     ${ssl.key.default};
+    ssl_verify_client       ${ssl.clientcertmode.default};
+    ssl_verify_depth        ${ssl.clientcertdepth.default};
+    ${web.ssl.dhparam.enabled}ssl_dhparam             ${web.ssl.dhparam.file};
+    return 444;
+}
+
 server
 {
-    ${core.ipboth.enabled}listen                  [::]:${web.https.port} default ipv6only=off;
-    ${core.ipv4only.enabled}listen                ${web.https.port} default;
-    ${core.ipv6only.enabled}listen                [::]:${web.https.port} default;
+    ${core.ipboth.enabled}listen                  [::]:${web.https.port} ipv6only=off;
+    ${core.ipv4only.enabled}listen                ${web.https.port};
+    ${core.ipv6only.enabled}listen                [::]:${web.https.port};
+
     ${web.add.headers.default}
-    server_name             ${web.server_name.default}.default;
+    server_name             ${web.server_name.default}; # add aliases and perhaps public
```

Comment:
I don't know if there is any significance to removing the .default from the server_name. I have tested with and without it, and it seems to be working just fine.